x-pack/solutions/observability/plugins/observability/dev_docs/rules_alerts_cases_overviewpage.md

```diff
@@ -0,0 +1,124 @@
+# Alerting
+
+## Testing
+
+1. Data generation 
```

Comment:
I didn't see step 2, maybe we can change it to a title? Should it be at the level of `Rules and alerts`?

Reply:
I updated the file structure. It should be better now.